app/models/service_ansible_playbook.rb

```diff
@@ -75,6 +75,9 @@ def get_job_options(action)
 
   def save_job_options(action, overrides)
     job_options = options.fetch_path(:config_info, action.downcase.to_sym).slice(:hosts, :extra_vars).with_indifferent_access
+    job_options[:extra_vars].try(:transform_values!) do |val|
+      val.kind_of?(String) ? val : val[:default] # TODO: support Hash only
```

Comment:
No. Since beta release every new service is having the new format. We don't need provide a migration for pre-beta versions.